### ISSUE TITLE
Error: invalid stored block lengths

### ISSUE THREAD

**Issue by polikin** [avocode/avocode#607]
I tried to upload a psd with the windows app.

psd size: 130kb.
os: windows 8.1

-----
App v.2.5.2
Source UID: 80425
```
Error: invalid stored block lengths
  at Zlib._handle.onerror (zlib.js:345:17)

```

**Comment by vojtatranta** [comment]
Hi, thanks for your report.
The problem is there because of cancelled downloading of parsed design, we will release fix soon, once we test it.
We are really sorry for that.

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
